test/test_autograd.py

```diff
@@ -1,54 +1,83 @@
 # Owner(s): ["module: autograd"]
 
+import collections
```

Comment:
I guess all the import shuffling is auto-formatting?
We usually don't do these unless they're from the lint to avoid shuffling back and forth depending on dev environments.

Reply:
ah whoops - i'll let lint runner check to see if this is ok, otherwise i'll make it original format

Reply:
reverted formatting changes